src/main/java/Duke.java

```diff
@@ -1,10 +1,134 @@
+import todo.Deadline;
+import todo.Event;
+import todo.Task;
+import todo.ToDo;
+
+import java.util.Scanner;
+
+//Check if valid command (Done, List, Bye, Action)
+//If done, mark as done
+//If action, check what action is it
+//If there is a date, what is the date
+
 public class Duke {
+
+    private static int currentIndex = 0;
+    private static int doneTask = 0;
```

Comment:
just to add on, the reasoning behind is that `doneTask` sounds more like a boolean (done or not done) rather than a counter for how many tasks have been completed.